src/components/dashboard/subscriptions/Subscription.js

```diff
@@ -1,38 +1,69 @@
 import React, {Component, PropTypes} from 'react';
-import {Link} from 'react-router';
-
 import Select from 'react-select';
 
+import BeanItemImage from '../browse/BeanItemImage';
+
 class Subscription extends Component {
-    render() {
-        const item = this.props.item;
-        const frequencyOptions = [
-            {value: 'MONTHLY', label: 'Monthly'},
-            {value: 'TRIMONTHLY', label: 'Trimonthly'},
-            {value: 'BIMONTHLY', label: 'Bimonthly'},
-            {value: 'WEEKLY', label: 'Weekly'}
+    constructor(props) {
+        super(props);
+
+        this.frequencyOptions = [
+            {value: 'MONTHLY', label: 'Every Month'},
+            {value: 'TRIWEEKLY', label: 'Every Three Weeks'},
+            {value: 'BIWEEKLY', label: 'Every Two Weeks'},
+            {value: 'WEEKLY', label: 'Every Weeks'}
         ];
-        const btnClass = 'pointer dim br1 ba bw1 tc pa2 black';
-        const statusLabel = item.status === 'ACTIVE' ? 'Deactivate' : 'Activate';
+    }
 
-        const date = new Date(item.createdAt);
+    render() {
+        const {item, bean} = this.props;
+
+        const btnClass = 'pointer dim br2 ba bw1 tc pa2 white bg-red';
+        const statusLabel = item.status === 'ACTIVE' ? 'Pause' : 'Activate';
+
+        const date = new Date(item.createdAt).toDateString();
+        const next = new Date(item.nextOrder).toDateString();
+
+        let bag = 'bag';
+        if (bean && item.quantity > 1) {
+            bag += 's';
+        }
```

Comment:
huge nit-pick to stir up controversy.  there's a ternary and incredible variable name opportunity:

```
const bagTag = bean && item.quantity > 1 ? 'bags' : 'bag'; 
```